### PR TITLE
Sort people by their start date

### PR DESCRIPTION
**Pull request by AndreasMadsen**
screencapture-127-0-0-1-4000-people-2021-09-09-15_49_59

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -37,12 +37,16 @@
   title: pi
   picture: sarath-chandar
   website: http://sarathchandar.in/
+  period:
+    start: 1970-01-01
```
Comment (msargordi) [inline _data/people.yaml:41]: What is this date? :D

File: _data/people.yaml [hunk 2]
```diff
@@ -37,12 +37,16 @@
...
 - name: Janarthanan Rajendran
   title: postdoc
   picture: janarthanan-rajendran
   website: https://sites.google.com/umich.edu/janarthanan-rajendran/
   cosupervisor: Doina Precup
+  period:
+    start: 2021-06-21
   topics:
     - en: Reinforcement Learning
       fr: Apprentissage profond par renforcement
```
Comment (msargordi) [inline _data/people.yaml:52]: I don't know French, but I think "profond" should be omitted, based on other translations.

File: _data/people.yaml [hunk 3]
```diff
@@ -57,6 +61,8 @@
   title: phd
   picture: vivswan-shitole
   website: https://viv92.github.io/
+  period:
+    start: 2021-09-06
```
Comment (msargordi) [inline _data/people.yaml:65]: I just could find the date to check it.

File: _data/people.yaml [hunk 4]
```diff
@@ -68,6 +74,8 @@
   picture: gabriele-prato
   website: https://pratogab.github.io/
   cosupervisor: Alain Tapp
+  period:
+    start: 2019-06-01
```
Comment (msargordi) [inline _data/people.yaml:78]: I think this one should be 2019-05-01, as the summer semester is May.

File: _data/people.yaml [hunk 5]
```diff
@@ -189,6 +217,8 @@
   title: master
   picture: pranshu-malviya
   website: http://pranshu28.github.io/
+  period:
+    start: 2021-01-01
```
Comment (msargordi) [inline _data/people.yaml:221]: Couldn't check one as well.

File: _data/people.yaml [hunk 6]
```diff
@@ -202,6 +232,8 @@
   picture: daphne-lafleur
   website: https://www.linkedin.com/in/daphné-lafleur-173a72170
   cosupervisor: Gilles Pesant
+  period:
+    start: 2021-06-01
```
Comment (msargordi) [inline _data/people.yaml:236]: She says the date is 28, but I guess it is not that important :D

File: _data/people.yaml [hunk 7]
```diff
@@ -226,6 +260,8 @@
   title: intern
   picture: paul-aymeric-mcrae
   website: https://aymeric-mcrae.github.io/
+  period:
+    start: 2020-09-01
```
Comment (msargordi) [inline _data/people.yaml:264]: Couldn't check it.

File: _data/people.yaml [hunk 8]
```diff
@@ -236,6 +272,8 @@
   title: intern
   picture: shabnam_haghzare
   website: https://www.shabnamhaghzare.com/
+  period:
+    start: 2021-05-11
```
Comment (msargordi) [inline _data/people.yaml:276]: couldn't check.

File: _data/people.yaml [hunk 9]
```diff
@@ -252,6 +290,8 @@
   title: intern
   picture: maziar-sargordi
   website: https://msargordi.github.io/
+  period:
+    start: 2021-05-01
   topics:
     - en: Machine Learning
       fr: Machine Learning
```
Comment (msargordi) [inline _data/people.yaml:297]: French translation is English.

File: _data/people.yaml [hunk 10]
```diff
@@ -262,6 +302,8 @@
   title: intern
   picture: yi-wan
   website: https://sites.google.com/ualberta.ca/yiwan/
+  period:
+    start: 2021-05-01
   topics:
     - en: Reinforcement Learning
       fr: Apprentissage profond par renforcement
```
Comment (msargordi) [inline _data/people.yaml:309]: I guess "profond" should be omitted.

**Review by msargordi — APPROVED** [review]
Hi Andreas, I tried to check all the dates. I did so, but I didn't know some of the date. They are mentioned in the comments. Also, some of the French translations seems to be wrong. Hope the comments are helpful.

File: _data/people.yaml (hunk 1 above)
Comment (AndreasMadsen) [reply _data/people.yaml:41]: The earliest possible date.

File: _data/people.yaml (hunk 2 above)
Comment (AndreasMadsen) [reply _data/people.yaml:52]: The translations were done by a native french speaker. Also, please only review the actual changes.

**Pull request merged**